spk/tvheadend/src/service-setup.sh

```diff
@@ -0,0 +1,128 @@
+# Package specific behaviors
+# Sourced script by generic installer and start-stop-status scripts
+
+# Service configuration. Change http and htsp ports here and in conf/tvheadend.sc for non-standard ports
+HTTPP=9981
+HTSPP=9982
+
+# Replace generic service startup, run service in background
+GRPN=`id -gn ${EFF_USER}`
+HOME_DIR="${SYNOPKG_PKGDEST}/var"
```

Comment:
What is that variable used for? By the way, it should be `HOME`, not `HOME_DIR`... probably the reason why you enforced location with patch

Reply:
@ymartin59 It's used in the `SERVICE_COMMAND`. The `-c` option sets the configuration directory. This is independent of the default recording directory, just coincides. I haven't been able to get it to work without the patch, despite defining `HOME` and using `ENV` in the `SERVICE_COMMAND`.